numpy/lib/function_base.py

```diff
@@ -728,12 +728,21 @@ def copy(a, order='K'):
         as possible. (Note that this function and :meth:`ndarray.copy` are very
         similar, but have different default values for their order=
         arguments.)
+    subok : bool, optional
+        If True, then sub-classes will be passed-through, otherwise the
+        returned array will be forced to be a base-class array (default).
```

Comment:
```suggestion
        returned array will be forced to be a base-class array (defaults to False).
```

Reply:
I took the wording here straight out of the `np.array` docstring. If we change it here it should be changed everywhere else the parameter description appears verbatim, which (according to grep) is `array`, `broadcast_arrays` and `broadcast_to`.

I think your wording is indeed more clear - let me know if you think all the doc strings should be modified or if we should leave it as-is.

Reply:
I think we should keep the change local. Making our use of "default" more consistent across the code base is a worthy goal, but a bit beyond this PR.